internal/wiz/structs.go

```diff
@@ -562,10 +562,13 @@ type AzureServiceBusAutomationActionParams struct {
 	QueueURL                string    `json:"queueUrl"`
 }
 
-// Connector struct -- updates.  this resource is incomplete.  missing ConnectorConfigs ConnectorModules TenantIdentityClient
+// Connector struct -- updates.  this resource is incomplete
+// not all ConnectorConfigs are defined, missing ConnectorModules TenantIdentityClient
 type Connector struct {
-	AddedBy           User            `json:"addedBy"`
-	AuthParams        json.RawMessage `json:"authParams"`
+	AddedBy           User            `json:"addedBy"` // this has been deprecated and should be replaced by CreatedBy
+	AuthParams        interface{}     `json:"authParams"`
```

Comment:
This shows as JSON in the GraphQL schema.  Can you explain why it is defined as an interface{} in the golang?

Reply:
was with experimenting to use a single connector for all CSPs, should have been cleaned up when dropping the `CreateConnectorAuthConfigAWSInput`